packages/pieces/community/wedof/src/lib/actions/send-file.ts

```diff
@@ -38,23 +38,23 @@ export const sendFile = createAction({
         required: false,
     }),
     typeId: Property.DynamicProperties({
```

Comment:
@obenazouz-wedof You can directly use Dropdown prop instead of DynamicProperties. As per [endpoint docs](https://www.wedof.fr/api/doc/operations/get_app_api_certificationfolder_listallcertificationfoldersfile) , use typeId as value and name as label.